### ISSUE TITLE
ChaCha20 support

### ISSUE THREAD

**Issue by tomato42** [trevp/tlslite#95]

**Comment by trevp** [comment]
yeah that draft's been stable for awhile it's unlikely to change, and major players like Google are going to implement, so it would be great to add support.
